<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenCamerBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseCamerBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveVideoBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenVideoBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开视频</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CutImageBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
